example/pyo3/run.py

```diff
@@ -53,21 +53,30 @@ def parse_args() -> Tuple[AccelerationMode, Path, str, Path]:
         help='モード ("AUTO", "CPU", "GPU")',
     )
     argparser.add_argument(
-        "open_jtalk_dict_dir",
+        "--dict",
+        default="./voicevox_core/open_jtalk_dic_utf_8-1.11",
         type=Path,
         help="Open JTalkの辞書ディレクトリ",
     )
     argparser.add_argument(
-        "text",
+        "--text",
+        default="この音声は ボイスボックスを使用して 出力されています。",
```

Comment:
別に大丈夫なのですが、特異なサンプルコードはなにか意図を感じられてしまうかもなので、念のため一般的な句読点のほうが良いかも
```suggestion
        default="この音声は、ボイスボックスを使用して、出力されています。",
```